### ISSUE TITLE
MacOS: Voiceover is unable to access options in the 'Add files and more' menu

### ISSUE THREAD

**Issue by ashleycox** [openai/codex#30708]
### What version of the Codex App are you using (From “About Codex” dialog)?

26.623.70822

### What subscription do you have?

Pro

### What platform is your computer?

Darwin 25.5.0 arm64 arm

### What issue are you seeing?

I'm a Voiceover screenreader user. In recent versions of the Codex desktop app on MacOS, I'm unable to access options in the 'Add Files and More' popup. When I activate the menu, voiceover announces that it is "expanded". As soon as I try to navigate the menu it changes to "collapsed" and I can't access any of the options. it used to pop up a menu that I could read like any other popup. Other popups, such as the settings, access or model popups work fine.

### What steps can reproduce the bug?

open codex desktop. turn on The Voiceover screenreader. Use ctrl+option+left and right arrows to land on the "add files and more" button. Press ctrl + option + space to press it. Then try to read the menu either using the up and down arrow keys, ctrl + option + up and down arrows, or ctrl + option + left and right arrows. If you press ctrl + option + left arrow immediately after pressing ctrl + option + space on the menu, you can sometimes 'see' some of the controls, but you can't activate them and the menu disappears.

### What is the expected behavior?

The menu opens and can be navigated using the arrow keys like any other menu.

### Additional information

it was working before in previous releases. I don't know when it failed precisely, I'd estimate some time in May, maybe early June.
